### ISSUE TITLE
1.4.4 Resize Text  - old content in the understanding

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag#3517]
When working on WCAG2ICT noticed the following in the 3rd paragraph of the Intent, bold and italics added by me:
>The author cannot rely on the user agent to satisfy this Success Criterion for HTML content if users do not have access to a user agent with zoom support. For example, if they work in an environment that requires them to use **_IE 6_**.

Not sure who uses IE 6 any more, as it reached end of support in July 2010.  The question remains though, are there modern browsers that still do not support zoom up to 200%? Just curious, as this statement may be able to be stricken. Otherwise, suggest removing the example.

**Comment by patrickhlauke** [comment]
Agree, this is ancient. Will make a PR to remove the entire paragraph, as nowadays full-page zoom is the norm.

**Comment by patrickhlauke** [comment]
https://github.com/w3c/wcag/pull/3559
